simplecnf.go

```diff
@@ -0,0 +1,160 @@
+package go_sat_solver
+
+import "math"
+
+type orClause []VariableAssignment
```

Comment:
not sure why an or clause is thought of as an array of assignments, rather than simply variables

Reply:
now that i understand the intention, it seems like these things should not be called variable assignments; they're more like "assignments for variables which will satisfy the or clause in which they occur"--desired variable assignments, or something.